flexget/plugins/cli/series.py

```diff
@@ -112,27 +130,40 @@ def display_summary(options):
         console(footer)
 
 
-def begin(manager, options):
+def manage(manager, options, update=False):
```

Comment:
Wait, what is even going on here? Why is the series plugin cli even being modified? I thought this series_list implementation was separate from series plugin?